pkg/controllermanager/controller/seed/seed_backup_bucket_check_control.go

```diff
@@ -57,41 +58,46 @@ func (c *Controller) backupBucketUpdate(oldObj, newObj interface{}) {
 		return
 	}
 
-	if !apiequality.Semantic.DeepEqual(oldBackupBucket.Status, newBackupBucket.Status) || !apiequality.Semantic.DeepEqual(oldBackupBucket.Spec, newBackupBucket.Spec) {
```

Comment:
Why did you change this condition?

Reply:
The reconciler only checks the `backupBucket.status.lastError` and `backupBucket.status.lastError.description` so I felt like there was no reason to enqueue it in case of other changes. Additionally, now the the reconciliation will be requeued based on the `syncPeriod` setting and I felt like there was no reason to additionally enqueue it only based on `backupEntry.spec` changes.